pkg/connector/slack/v0/config/definition.json

```diff
@@ -23,12 +23,13 @@
               "string"
             ],
             "instillCredentialField": false,
+            "instillSecret": true,
             "instillUIOrder": 0,
             "title": "token",
             "type": "string"
           }
         },
-        "required": [],
+        "required": ["token"],
         "title": "Slack Connection",
```

Comment:
Add additional `"instillEditOnNodeFields": ["token"]` to make it appears on the Console node.